src/modules/AdvancedPaste/AdvancedPaste/ViewModels/OptionsViewModel.cs

```diff
@@ -291,15 +311,28 @@ internal void ToJsonFunction(bool pasteAlways = false)
             try
             {
                 Logger.LogTrace();
+                string tempText = ClipboardData.GetTextAsync().GetAwaiter().GetResult();
+                Logger.LogInfo("Temp text: " + tempText);
```

Comment:
remove all the logging used for testing here. (obsolete as we log in clipboard helper.)